### DISCUSSION TITLE
Turn off error popup

### DISCUSSION THREAD

**Discussion by jouzie** [dbeaver/dbeaver#17450] [Q&A]
<img width="431" alt="Screenshot 2022-08-24 at 11 25 27" src="https://user-images.githubusercontent.com/111965187/186369443-9eb485ba-a067-4132-a271-e2be3a669fca.png">

How to turn off this error popup and  make it so the error message appears together with the results

**Comment by LonwoLonwo** [comment]
Hello @jouzie 

Please check this setting:
Preferences -> Editors -> Data Editor -> Show errors in a dialog
And try to disable it if it is enabled. 

![2022-08-26 21_39_43-DBeaver Enterprise 22 1 3 - _postgres_ Script-12](https://user-images.githubusercontent.com/45152336/186970915-8eba91e0-1800-476d-a6aa-2cf8dcc939ee.png)

**Reply by jouzie** [reply]
Unfortunately pop-up with the error still persists
